rust/ql/lib/codeql/rust/internal/Type.qll

```diff
@@ -320,6 +333,55 @@ class TypeParamTypeParameter extends TypeParameter, TTypeParamTypeParameter {
   }
 }
 
+/** Gets type alias that is the `i`th type parameter of `trait`. */
```

Comment:
`Gets type` -> `Gets the type`.

Reply:
Perhaps add a comment that type aliases are numbered arbitrarily but consecutively, starting from the index following the last ordinary type parameter.